### PR TITLE
Working on the routes to create a ticket.

### PR DESCRIPTION
**Pull request by jessampaio**
I have to think about how I'm going to structure my SQL database before I fix the routes. I was thinking to write a query to join the tickets table with the events table by the event id and then check how many tickets are left to be created for that event by summing the ticket_quantity and subtracting from the ticket_amount on the event table.

I am not sure yet. Going to work on that this week.

### CODE REVIEW

File: express-typescript/src/controllers/tickets.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import database from '../database'
+import { type Request, type Response } from 'express'
+
+export function getPricedTickets(req: Request, res: Response) {
+  const getPricedTicketsQuery = `SELECT * FROM tickets`
```
Comment (herlitzj) [inline express-typescript/src/controllers/tickets.ts:5]: This is just going to return the entire table. Is that your intent?

File: express-typescript/src/controllers/tickets.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    
+  const createTicketQuery = `INSERT INTO tickets (
+      ticket_price, 
+      event_id, 
+      ticket_tier,
+      ticket_quantity
+    ) VALUES (?)`
+
+  const values = [
+    req.body.ticket_price,
```
Comment (herlitzj) [inline express-typescript/src/controllers/tickets.ts:30]: Request body should probably be camelCase

File: express-typescript/src/controllers/tickets.ts [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    req.body.ticket_tier,
+    req.body.ticket_quantity,
+  ]
+
+  database.query(createTicketQuery, [values], (err, data) => {
+    if (err != null) {
+      console.log(err)
+      return res.status(500).json(err)
+    }
+    return res.status(200).send('Ticket has been added successfully.')
```
Comment (herlitzj) [inline express-typescript/src/controllers/tickets.ts:41]: Don’t need to specify 200. That’s the default

File: express-typescript/src/controllers/tickets.ts [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  })
+}
+
+export function getTicket (req: Request, res: Response) {
+  const getTicketQuery = `SELECT * FROM events WHERE event_id = ?`
```
Comment (herlitzj) [inline express-typescript/src/controllers/tickets.ts:46]: Why is a function called getTicket selecting from the events table?

**Review by herlitzj — APPROVED** [review]
Some minor questions but seems ok

**Comment by jessampaio** [comment]
I will fix it. I was just testing to see how to push a branch to the remote repo :D

**Pull request merged**